README.md

```diff
@@ -68,6 +69,7 @@ The `buildCache` configuration block might look like this:
          region = 'eu-west-1'
          bucket = 'your-bucket'
          push = isCiServer
+         headers = [ 'x-header-name': 'header-value' ]
```

Comment:
Please remove this. The sample config should be as clean as possible. If you think, the option needs further explanation, please add an additional section where you explain the usage in more detail.